azure/template.json

```diff
@@ -509,34 +509,6 @@
                 "name": "SETTINGS__AUTHENTICATION__SECRET",
                 "value": "[parameters('settingsAuthenticationSecret')]"
               },
-              {
```

Comment:
I think this one needs to remain

Reply:
From the trello ticket:
>Additionally, `SETTINGS__GOVUK_NOTIFY__API_KEY` is only required in the `environmentVariables` section as it's only used in the background mailer process. It can be removed from the `appServiceAppSettings` section.